packages/@eventual/core/src/entity/entity.ts

```diff
@@ -565,6 +565,15 @@ export interface EntityQueryOptions extends EntityReadOptions {
   nextToken?: string;
 }
 
+export interface EntityQueryOptions extends EntityScanOptions {
+  /**
+   * When true, returns the items in the descending sort order.
+   *
+   * @default false - ascending order
+   */
+  sortDescending?: boolean;
+}
```

Comment:
Not sure if I like the name of this property.

What about:
```ts
order: "asc" | "desc"
```

Dynamo has ScanIndexForward which is also really don't like lol.

Reply:
I used `direction` to match our system APIs (`sortDirection`) and how postgres describes the ASC/DESC (https://www.postgresql.org/docs/current/queries-order.html).

> Each expression can be followed by an optional ASC or DESC keyword to set the sort direction to ascending or descending.

Dynamo does say "order", but I thought that would be confused with the key that determines the item order. :shrug:

Reply:
Direction doesn't make sense to me either. I don't see any reference to that in postgres docs. And I've never said "return them in ascending digestion". What was wrong with my suggestion of "order"?

Reply:
I'm surprised by the diversity of choices across databases haha

Reply:
Options:
```
sort: "asc"

order: "asc"

direction: "asc"
```

"direction" aligns with the way queries work (by scanning an index in a specific direction). But it feels off in some ways ..

Reply:
I'm fine with either order or direction. I paused on order because it isn't a field to order or sort on, but the direction to explore the results.

Reply:
Let's go with it